src/collection/components/CollectionOrBundleEditMetaData.tsx

```diff
@@ -53,17 +52,51 @@ const CollectionOrBundleEditMetaData: FunctionComponent<CollectionOrBundleEditMe
 	const [descriptionLongEditorState, setDescriptionLongEditorState] = useState<
 		RichEditorState | undefined
 	>(undefined);
+	const [lomFields, setLomFields] = useState<LomFieldsByScheme>({
+		contexts: [],
+		educationLevels: [],
+		subjects: [],
+		themes: [],
+	});
 
 	const isCollection = type === 'collection';
 
-	const updateCollectionMultiProperty = (
-		selectedTagOptions: TagInfo[],
-		collectionProp: keyof Avo.Collection.Collection
-	) => {
+	useEffect(() => {
+		if (!isEmpty(collection) && collection.loms) {
```

Comment:
you should only use isEmpty for checking arrays or empty objects, not for null/undefined checks. You can use isNil for that.